bitcoin/Cargo.toml

```diff
@@ -70,6 +70,14 @@ required-features = ["std", "rand-std"]
 name = "ecdsa-psbt"
 required-features = ["std", "bitcoinconsensus"]
 
+[[example]]
+name = "sign-tx-segwit-v0"
+required-features = ["std", "rand-std"]
+
+[[example]]
+name = "sign-tx-taproot"
+required-features = ["std", "rand-std"]
```

Comment:
Isn't these solved with only `"rand-std"` as for #2102?

Reply:
Yes, the "std" is redundant. I figured the explicitness didn't hurt so I just went with uniformity.